src/include/stir/Bin.h

```diff
@@ -79,6 +79,12 @@ class Bin
   
   //! accumulate voxel's contribution during forward projection 
   inline Bin&  operator+=(const float dx);
+  //! multiply bin values during normalisation -- apply()
```

Comment:
ok to add these, but remove the comments on apply/undo. It only makes sense for people who think about BinNormalisationFromProjData...
maybe should then add a subtract as well for consistency.